build/quickstatements/dockerhub.md

```diff
@@ -1,4 +1,4 @@
-# Wikibase Suite QuickStatements Image
+# Wikibase Suite quickstatements Image
```

Comment:
```suggestion
# Wikibase Suite QuickStatements Image
```
I think here we can keep camel case... As a product name in text.

Reply:
Agreed.